plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.inc.php

```diff
@@ -78,11 +78,7 @@ public function extractMetadataFromDataObject(&$publicationFormat)
         $dc11Description = $this->instantiateMetadataDescription();
 
         // Title
-        $titles = [];
-        foreach ($monograph->getTitle(null) as $titleLocale => $title) {
-            $titles[$titleLocale] = $monograph->getFullTitle($titleLocale);
-        }
-        $this->_addLocalizedElements($dc11Description, 'dc:title', $titles);
+        $this->_addLocalizedElements($dc11Description, 'dc:title', $monograph->getFullTitle(null));
```

Comment:
This makes use of the deprecated `Monograph::getFullTitle` function; you should be able to use instead:
```PHP
$publication = $monograph->getCurrentPublication();
$this->_addLocalizedElements($dc11Description, 'dc:title', $publication->getFullTitles());
```
Could you try that and adjust the PR? (Thanks for spotting this!)

Reply:
Hi Alec, thanks, that works, too. I made a new commit.